observations/README.md

```diff
@@ -0,0 +1,1279 @@
+| valueQuantity                                    | `ObservationStatement / value` where `ObservationStatement / value [@type]` is `"PQ"` or `"IVLPQ"`                                                       |
+| valueQuantity.unit                               | `ObservationStatement / value [@unit]` or `ObservationStatement / value / (high or low)`                                                                 |
+| valueQuantity.comparator                         | `ObservationStatement / value / (high or low) [@inclusive]` where `ObservationStatement / value [@type]` is `"IVLPQ"`                                    |
+| valueQuantity.extension\[0].url                  | fixed value = `"https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-ValueApproximation-1"` <sup>1</sup>                               |
+| valueQuantity.extension\[0].value                | fixed value = `true` <sup>1</sup>                                                                                                                        |
+| valueString                                      | `ObservationStatement / value` where valueQuantity is not populated                                                                                      |
+| effectiveDateTime                                | `ObservationStatement / effectiveTime` or else `ObservationStatement / availibiltyTime [@value]`                                                         |
+| issued                                           | `ehrCompostion / author / time [@value]` <sup>2</sup> or else `EhrExtract / availibilityTime [@value]`                                                   |
+| performer\[0]                                    | Practitioner referenced in `ObservationStatement / paticipant` <sup>3</sup>                                                                              |
+| interpretation.coding\[0].code                   | `ObservationStatement / interpretationCode [@code]`                                                                                                      |
+| interpretation.coding\[0].display                | `ObservationStatement / interpretationCode [@code]`                                                                                                      |
+| interpretation.coding\[0].system                 | fixed value = `"http://hl7.org/fhir/v2/0078"`                                                                                                            |
+| interpretation.text                              | `ObservationStatement / interpretationCode / originalText` or else `ObservationStatement / interpretationCode [@displayName]`                            |                                         
+| comment                                          | Concatenated from `ObservationStatement / subject / personalRelationship / code` and `ObservationStatement / pertinentInformation / pertinentAnnotation` |
```

Comment:
Minor: It appears to add in `ObservationStatement / code`. I don't know how we can be clear `ObservationStatement / pertinentInformation` is multiple elements. 🤔

Reply:
I've updated it with "each" and in the sequence defined by `ObservationStatement / pertinentInformation / sequenceNumber`